### DISCUSSION TITLE
Instalando o homebrew

### DISCUSSION THREAD

**Discussion by Krishnotsava** [Homebrew/discussions#5659] [Getting started]
### Output of `brew config`

```shell
/bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
```

### Output of `brew doctor`

```shell
Warning: /opt/homebrew/bin is not in your PATH.
  Instructions on how to configure your shell for Homebrew
  can be found in the 'Next steps' section below.
==> Installation successful!
```

### Description of issue

Estou iniciando um curso de python e fui configurar o meu terminal, no momento que fiz a instalação do homebrew: /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"

Ele instalou tudo certo, porém apareceu o seguinte Warning: 
 /opt/homebrew/bin is not in your PATH.
  Instructions on how to configure your shell for Homebrew
  can be found in the 'Next steps' section below.

**Comment by SMillerDev** [comment]
Not sure what the question is

**Comment by p-linnane** [comment]
Run the commands provided below as the message says.
